File: Arcade/Intro/C++/22_avoidObstacles.cpp
int avoidObstacles(std::vector<int> inputArray) {
    int jump=1;
    for(int i=0;i<inputArray.size();i++){
        if(inputArray[i]%jump==0){
            jump++;
            i=-1;
        }
    }
    return jump;
}

File: Arcade/Intro/JavaScript/22_avoidObstacles.js
function avoidObstacles(inputArray) {
    inputArray.sort();
    var jumps = 2;
    for (var i = 0; i < inputArray.length; i++) {
        if (inputArray[i] % jumps == 0) {
            jumps++;
            i = 0;
        }
    }
    return jumps;
}
